### ISSUE TITLE
A proto with import couldn't be properly processed

### ISSUE THREAD

**Issue by chundongwang** [bojand/grpc-inspect#8]
With https://github.com/bojand/grpc-inspect/pull/7 `grpc-inspect` starts to inspect packageName. However, if there exists `import "common.proto";` with another namespace, the logic would treat namespace as message then none of the services would be recognized.
Sample proto:
```
// common.proto
syntax = "proto3";
package common;
message Animal {
  string name = 1;
}
```

```
// human.proto
syntax = "proto3";
package human;
message Person {
  string last_name = 1;
}
message MakeHumanRes {
  string last_name = 1;
}
service HumanService {
  rpc MakeHuman() returns (MakeHumanRes);
}
```

**Comment by chundongwang** [comment]
This broke `mali` as `mali` just say `~0.3.0` for its dependency.

**Comment by bojand** [comment]
Fixed in #9.
